.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 node_js:
-  - '7'
+  - '8'
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH="$HOME/.yarn/bin:$PATH"
```

Comment:
Why is this needed?

Reply:
Travis uses yarn@0.27, this is required to update it to the latest. I copied it from [the Jest repo](https://github.com/facebook/jest/blob/5b2b647345a256a842f99f60ebc1eb843448ecc2/.travis.yml).